spec/formatting.md

```diff
@@ -505,14 +505,17 @@ _Formatting_ is a mostly implementation-defined process,
 as it depends on the implementation's shape for resolved values
 and the result type of the formatting.
 
-Formatting errors MAY be emitted during _formatting_,
-as formatting is not necessarily defined on every resolved value.
-A formatter MAY provide a value to use in such a case instead of a _fallback value_.
+Formatting is not necessarily defined on every resolved value.
+An implementation SHOULD emit formatting errors during _formatting_,
+but MAY provide a value to use in such a case instead of a _falback value_.
```

Comment:
You're right that the proposed normative sentence doesn't make sense.

The problem with the original sentence is that it is not usefully normative: "MAY" is trying to permit formatting errors to be emitted during formatting, but when else would formatting errors occur if not during formatting? As a direction to implementers it's not useful and it's not testable.

My proposal, of course, is worse, because it seems to imply that every formatting operation ought to produce errors 😭.